test/lib/BedrockTester.cpp

```diff
@@ -553,4 +553,38 @@ int BedrockTester::waitForPort(int port) {
     return 1;
 }
 
+// get the output of a "Status" command from the command port
+STable BedrockTester::getStatus( bool control ) {
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    SData request( "Status" );
+    string response = executeWaitVerifyContent( request, "200", control );
+    STable responseTable = SParseJSONObject(response);
+    return responseTable;
+}
+
+// get the value of a particular term from the output of a "Status" command
+string BedrockTester::getStatusTerm( string term, bool control ) {
```

Comment:
this function name cloud be more clear, `waitForStatusTerm`, `get` implies it will access once and return, not loop forever.

Reply:
I'd like to keep it. It's not expected to wait for some eventual state, it's meant to get something that is _usually, but not always_ there.